### PR TITLE
Events rehaul

### PR DESCRIPTION
**Pull request by wendi-yu**
Added aviation fun days, updated school outreach events to be more current, renamed "events" to "outreach" to be a bit clearer about the purpose of these events

### CODE REVIEW

File: outreach.html [hunk 1]
```diff
@@ -24,12 +24,46 @@
...
+                <li><a href="outreach">OUTREACH</a>
                 <li><a href="contact">CONTACT</a>
             </ul>
         </div>
         <div id="event_cover"></div>
-        <div class="event__wrapper">
+        <div class="outreach__info">
+            <h1 class="content-title">OUTREACH</h1>
+            <p class="content-text">
+                Waterloo Rocketry is a design team consisting entirely of students, and we'd love to share our work with the community around us! Outreach remains an important part of our work, as we seek to inspire the future generations of rocket scientists. Below are some events we've put on in the past, both within the University of Waterloo and in the wider Kitchener-Waterloo region. If you'd be interested in coordinating an outreach event with us, feel free to reach out to us at <a href="mailto:contact@uwaterloorocketry.com">contact@uwaterloorocketry.com</a>!
```
Comment (jacobdeery) [inline outreach.html]: "and we'd love" --> "and we love"

"Outreach remains an important part of our work as we seek to inspire future generations of rocket scientists."

File: outreach.html [hunk 2]
```diff
@@ -55,10 +89,13 @@ <h1 class="content-title">MODEL ROCKET COMPETITION</h1>
...
+                            The ESQ (Engineering Science Quest) day camp for children is run throughout the summer at University of Waterloo. Children participate in a number of science related actives. One of the activities is to visit with student engineering teams. Over the past years, we have run outreach sessions for over 250 children, most recently in 2018. We have taught them some of the basic principles of rocket design and flown small model rockets with them.  
+                        </p>
+                        <p>
+                            We've also coordinated outreach events with individual groups in the past, tailoring the depth of content we cover to the level of the students we visit. Specific examples include high school presentations, where we showcased <a href="vidar3.html">Vidar III</a>, and intro to rocketry tutorials for local Air Cadet squadrons.
```
Comment (jacobdeery) [inline outreach.html]: `<a href="vidar3">`

File: outreach.html [hunk 3]
```diff
@@ -24,12 +24,46 @@
...
+                    </div>
+                </div>
+                <div class="col-sm-4">
+                    <img class="poster" src="images/AviationFunDay_logo.jpg"/>
+                </div>
+            </div>
+            <div id="event__line" class="line"></div>
             <div class="event">
                 <div class="col-sm-8">
                     <h1 class="content-title">MODEL ROCKET COMPETITION</h1>
```
Comment (jacobdeery) [inline outreach.html:69]: It might be good to replace the poster in this section while we're at it. We're still using one with the old logo.

**Comment by wendi-yu** [comment]
Accidentally based the events-rehaul branch off a different branch (subsystems-page) rather than gh-pages, force pushed a rebase to fix the issue

**Review by jacobdeery — COMMENTED** [review]
These sections are giving me scrollbars:

image

**Comment by jacobdeery** [comment]
I'm also still seeing a lingering `subsystems.html` here

**Comment by wendi-yu** [comment]
Can I ask how you're getting the scroll bars? I checked through all the modes on developer tools and zoomed in but I can't seem to get them to display on mine

**Comment by jacobdeery** [comment]
Might be a browser thing. They only show up on Vivaldi for me at a certain zoom level (if I zoom in or out too far they go away), and not at all on Chrome. Looking at the live site, I'm getting them there as well, so this isn't a regression. Probably not important - who uses Vivaldi anyway?

**Review by jacobdeery — APPROVED** [review]

**Pull request merged**